package.json

```diff
@@ -119,6 +119,11 @@
                     },
                     "description": "Extra clang flags used to parse files when no compilation database is found."
                 },
+                "clangd.suppressCompletionReranking": {
```

Comment:
I think the doulbe-negative feel (suppress + "reranking" implying undoing the old ranking) makes this a bit hard to understand. And the description only mentions one half of the tradeoff, so it's hard to make a choice.

What about `clangd.serverCompletionRanking` / "Always rank completion items on the server as you type. This produces more accurate results at the cost of higher latency than client-side filtering."

Reply:
Thanks, I did not like the original name as well =) Fixed.